recipes/firefox-unbranded/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "firefox-unbranded" %}
+{% set version = "76.0.1" %}
+{% set tag = "e2de5f11bc0afd9a3024d32b83cb9f0ada95717a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.mozilla-release.revision.{{ tag }}.firefox.linux64-add-on-devel/artifacts/public/build/target.tar.bz2  # [linux64]
+  # Checksums available at:
+  # https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.mozilla-release.revision.{{ tag }}.firefox.linux64-add-on-devel/artifacts/public/build/target.checksums
+  sha512: a855df81b90a39fa4746fc2f691433eb2f0d75f4a0347fd1e937f221ef8265c9e88fb5fc3a63865c59eb6d60192b94aa83c4c186c68f101eef3f071ed613608f  # [linux64]
+
+build:
+  skip: True  # [not linux64]
+  number: 0
+  binary_relocation:
+    - bin/firefox
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  # Unfortunately can't test on linux due to conda-forge build infrastructure 
+  # not having gtk3. https://github.com/conda-forge/staged-recipes/issues/9314
```

Comment:
We have `gtk3` now.

Reply:
If we can solve it https://github.com/conda-forge/firefox-feedstock/pull/18 I'll fix it here too.